helm/ffc-demo-payment-service-core/values.yaml

```diff
@@ -53,17 +53,17 @@ readinessProbe:
   path: /healthy
   port: 3007
   initialDelaySeconds: 60
-  periodSeconds: 20
+  periodSeconds: 60
   failureThreshold: 3
-  timeoutSeconds: 30
+  timeoutSeconds: 60
 
 livenessProbe:
   path: /healthz
   port: 3007
   initialDelaySeconds: 70
-  periodSeconds: 20
+  periodSeconds: 60
```

Comment:
why 60 secs?

Reply:
It was increased during debugging. Changing it back to original value of 20seconds